src/vcpkg-test/manifests.cpp

```diff
@@ -1024,3 +1021,43 @@ TEST_CASE ("Serialize all the ports", "[all-manifests]")
         REQUIRE(*serialized_scf == scf);
     }
 }
+
+static Json::Object manifest_with_license(StringView license)
```

Comment:
parse_spdx_license should be unit tested itself not only through manifest parsing.

Reply:
could you relook?